src/numbers.js

```diff
@@ -1,45 +1,45 @@
 const add = (a, b) => {
-  // your code here
+  return a + b 
 };
 
 const subtract = (a, b) => {
-  // your code here
+  return a - b
 };
 
 const multiply = (a, b) => {
-  // your code here
+  return a * b
 };
 
 const divide = (a, b) => {
-  // your code here
+  return a / b
 };
 
 const power = (a, b) => {
-  // your code here
+  return Math.pow (a, b)
```

Comment:
When we define functions and methods, we use spaces around the parentheses, but when we call / invoke them (that is, we use them), we don't use spaces, so:

```js
return Math.pow(a, b);
```